easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.4.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'LibTIFF'
+version = '4.4.0'
+
+homepage = 'https://libtiff.maptools.org/'
```

Comment:
This is not loading for me. How about https://libtiff.gitlab.io/libtiff/ instead?

Reply:
See https://github.com/boegel/easybuild-easyconfigs/pull/90